.changeset/easy-vans-laugh.md

```diff
@@ -0,0 +1,7 @@
+---
+'astro': minor
+---
+
+Updates when the configuration is validated to include integrations
```

Comment:
Is this of interest/importance only to integration authors? If so, it's probably helpful to make that clear so that "regular users" with an Astro project aren't wondering whether this affects them. Like, is it something like:

```suggestion
Updates Astro config schemas and the Integrations API lifecycle hooks that determine when the configuration is validated. Also adds error logging specifying which integration/hook is failing validation.
```

or something like this that can be more explicit about this affecting integration authoring? (assuming that's the case!)

Reply:
```suggestion
Updates Astro config validation to also run for integrations. An error log will specify what integration is failing the validation
```
Wdyt?

Reply:
I think that's good! But it still is unclear whether I should care if I'm a *user* of integration, or an *author* of integrations. (That's why I had mentioned the Adapter API originally, figuring that people who don't use that would know that they can tune out, if that's the case.) That's kind of my main thing with both these comments, that *everyone* will read these messages and it needs to be clear who they are for! Most people have *installed* integrations, so just knowing something is "about integrations" isn't enough to make it clear.

Reply:
Updated!

Reply:
Looks great, thank you!